apps/package.json

```diff
@@ -213,7 +213,7 @@
     "uglifyjs-webpack-plugin": "2.1.1",
     "unminified-webpack-plugin": "^2.0.0",
     "url-loader": "^0.5.7",
-    "video.js": "4.5.2",
+    "video.js": "7.6.6",
```

Comment:
This is a big jump.  Is there anything else that depends on this version or that this version depends on? I don't know what it might be, just that when we bump versions of things it has the potential to be problematic.